gh-cli/README.md

```diff
@@ -737,6 +737,45 @@ Sets the IP allow list to enabled/disable for an enterprise or organization. You
 
 See the [docs](https://docs.github.com/en/graphql/reference/mutations#updateipallowlistenabledsetting) for further information.
 
+## set-org-ip-allow-list-rules.sh
+
+Sets the IP allow list rules for an organization from a set of rules defined in a file, the script is idempotent running it multiple times will only make the changes needed to match the rules in the file.
+
+In order to insure availability of the service, the script first adds all necessary rules and only after that will delete rules no longer applicable, this insures no disruption of service if the change has an (partial) overlapping set of rules.
+
+Optionally you can opt-in in to save a backup of rules on GitHub before the changes are applied.
+
+> **Warning**
+> The script doesn't take into account if existing rules are active, if changes are made to an inactive rule it will be become active. If no changes are made, then active status will be ignored.
```

Comment:
```suggestion
```suggestion
Sets the IP allow list rules for an organization from a set of rules defined in a file. The script is idempotent; running it multiple times will only make the changes needed to match the rules in the file.

In order to ensure availability of the service, the script first adds all necessary rules and only after that will delete rules no longer applicable. This ensures no disruption of service if the change has an (partial) overlapping set of rules.

Optionally, you can opt-in in to save a backup of rules on GitHub before the changes are applied.

> **Warning**
> The script doesn't take into account if existing rules are active. If changes are made to an inactive rule it will be become active. If no changes are made, then active status will be ignored.
```